README.rst

```diff
@@ -0,0 +1,260 @@
+
+PIQ was initially named `PhotoSynthesis.Metrics <https://pypi.org/project/photosynthesis-metrics/0.4.0/>`_
+
+.. contents:: Table of Contents
+
+Installation
+------------
+`PyTorch Image Quality (PIQ) <https://github.com/photosynthesis-team/piq>`_ can be installed using ``pip``, ``conda`` or ``git``.
+
+
+If you use ``pip``, you can install it with:
+
+.. code-block:: sh
+    
+    $ pip install piq
+
+
+If you use ``conda``, you can install it with:
+
+.. code-block:: sh
+    
+    $ conda install piq -c photosynthesis-team -c conda-forge -c PyTorch
+
+
+If you want to use the latest features straight from the master, clone `PIQ repo <https://github.com/photosynthesis-team/piq>`_:
+
+.. code-block:: sh
+
+   git clone https://github.com/photosynthesis-team/piq.git
+   cd piq
+   python setup.py install
+
+
+Documentation
+-------------
+
+The full documentation is available at https://piq.readthedocs.io.
+
+Usage Examples
+---------------
+
+Image-based metrics
+^^^^^^^^^^^^^^^^^^^
+The group of metrics (such as PSNR, SSIM, BRISQUE) takes image or images as input.
+We have a functional interface, which returns a metric value, and a class interface, which allows to use any metric
+as a loss function.
+
+.. code-block:: python
+
+   import torch
+   from piq import ssim, SSIMLoss
+
+   x = torch.rand(4, 3, 256, 256, requires_grad=True)
+   y = torch.rand(4, 3, 256, 256)
+
+   ssim_index: torch.Tensor = ssim(x, y, data_range=1.)
+
+   loss = SSIMLoss(data_range=1.)
+   output: torch.Tensor = loss(x, y)
+   output.backward()
+
+For a full list of examples, see `image metrics <https://github.com/photosynthesis-team/piq/blob/master/examples/image_metrics.py>`_ examples.
+
+Feature-based metrics
+^^^^^^^^^^^^^^^^^^^^^
+
+The group of metrics (such as IS, FID, KID) takes a list of image features.
+Image features can be extracted by some feature extractor network separately or by using the ``compute_feats`` method of a
+class.
+
+Note:
+    ``compute_feats`` consumes a data loader of a predefined format.
+
+.. code-block:: python
+
+   import torch
+   from torch.utils.data import DataLoader
+   from piq import FID
+
+   first_dl, second_dl = DataLoader(), DataLoader()
+   fid_metric = FID()
+   first_feats = fid_metric.compute_feats(first_dl)
+   second_feats = fid_metric.compute_feats(second_dl)
+   fid: torch.Tensor = fid_metric(first_feats, second_feats)
+
+
+If you already have image features, use the class interface for score computation:
+
+.. code-block:: python
+
+    import torch
+    from piq import FID
+
+    x_feats = torch.rand(10000, 1024)
+    y_feats = torch.rand(10000, 1024)
+    msid_metric = MSID()
+    msid: torch.Tensor = msid_metric(x_feats, y_feats)
+
+
+For a full list of examples, see `feature metrics <https://github.com/photosynthesis-team/piq/blob/master/examples/feature_metrics.py>`_ examples.
+
+
+List of metrics
+---------------
+
+Full Reference
+^^^^^^^^^^^^^^
+
+===========  ======  ==========
```

Comment:
The tables with list of metrics should be imported as csv files similarly to benchmark to simplify the reuse in different places.